test/runtests.jl

```diff
@@ -75,8 +75,8 @@ end
 @testset "  ex111: Chebyshev example                   " begin
     include("../examples/ex111_chebyshev.jl")
 
-    @test minimum(eigenvalues) ≈ -0.272875736279538
-    @test maximum(eigenvalues) ≈ 0.26048711552603726
+    @test minimum(eigenvalues) ≈ -0.2301200211798214
+    @test maximum(eigenvalues) ≈ 0.1306886015364882
```

Comment:
I'd like to check that both the old behavior is retained and the new behavior works as intended here. Could we add a `ex112_chebyshev_fourth.jl` and keep this one as is?

Reply:
Sure -- done!